Objective-C/CBLDatabase.h

```diff
@@ -590,7 +590,7 @@ __deprecated_msg("Use [[database defaultCollection] getDocumentExpirationWithID:
         default collection is deleted.
  @return Default collection, or nil if an error occurred.
  */
-- (nullable CBLCollection*) defaultCollection: (NSError**)error;
+- (nullable CBLCollection*) defaultCollection: (NSError**)error NS_SWIFT_NOTHROW;
```

Comment:
There is no case as of now to return a null default collection without an error.